docs/projects/README.md

```diff
@@ -0,0 +1,469 @@
+- Identify areas of technical ambiguity that cannot be predetermined during
+  planning (i.e., after we've implemented _x_, we'll be able to make an informed
+  decision about _y_).
+- Identify atomic blocks of work so that work can be split into individual,
+  small, easily reviewable PRs
+  - Ideally PRs can be reviewed by anyone capable of reviewing in the relevant
+    parts of the codebase, not only people intimately familiar with the project
+  - This isn't always possible, but it is good to strive for it as much as we
+    can
+- Any milestones. Moments when significant or discrete chunks of work have been
+  completed, or sub-features can ship.
+- Any accessibility concerns or requirements.
+- Rollback
+  - How do we rollback this solution in the event of failure?
+  - Are there any steps that can not easily be rolled back?
+- Privacy
+  - How does this approach protect users' privacy?
+- Localization
+  - Any translation or regional requirements?
+  - Any differing legal requirements based on user location?
+- Risks
+  - What risks are we taking with this solution?
+  - Are there risks that once taken can’t be undone?
+- Prior art
+  - Include links to documents and resources that you used when coming up with
+    your solution.
+  - Credit people who have contributed to the solution that you wish to
+    acknowledge.
+
+> **Note**
+>
+> We do not have any recommendations about time estimates in this section.
+
+#### The review process
+
+Technical implementation plans will be open for a two-week review period. In
+cases where relevant contributors are AFK or otherwise unavailable, or a
+implementation plan experiences low engagement for a number of reasons,
+implementation plan deadlines will be extended by one week at a time as
+necessary.
+
+#### Wrapping up
+
+Finally, the implementation plan is merged into the repo. Project contributors
+create GitHub issues for all of the work identified in the implementation plan.
+This process can be quite time-consiming for large projects. Generally, it makes
+sense to create issues in the order which they must be completed; this allows
+work to begin if, for some reason, there is delay in creating issues for work at
+the end of a project. It also however makes sense to priorize issues that are
+"good first issues" and "help wanted" issues contributors outside of the core
+maintainers can help with.
+
+It is important to make sure dependencies are documented when creating these
+issues. Issues which depend on another issue or set of issues should be labeled
+as "blocked" and reference the blocking issue(s).
+
+Whenever possible, issues should be written so that an implementer can complete
+an atomic unit of work without needing to understand the full scope and
+technicalities of the project. The issue description should contain all
+necessary information to complete the issue. This can include linking to
+specific relevant sections of the existing implementation plan(s).
```

Comment:
@sarayourfriend here's the new section with advice on writing granular, standalone issues.